constants/index.ts

```diff
@@ -453,3 +453,20 @@ export const inputFields = [
     marginTop: true,
   },
 ];
+
+export const laptopMockupStyles = {
+  homepageParent: "h-40 w-64 lg:h-[21rem] lg:w-[34.5rem]",
+  otherPageParent: "h-[10.6rem] w-[17rem] sm:h-[19.5rem] sm:w-[32.35rem]",
```

Comment:
We'll slowly get better at naming our variables. :)

Reply:
@cdsalty Man it's so hard. I'm sure someone out there could make good money being a professional variable namer